spec/spec.html

```diff
@@ -118,7 +118,7 @@ <h1 class="title">The Austral Language Specification</h1>
           <li><a href="#stmt-let" id="toc-stmt-let">Let
           Statement</a></li>
           <li><a href="#stmt-let-destructure"
-          id="toc-stmt-let-destructure">Let Destructure
+          id="toc-stmt-let-destructure">Let-destructure
```

Comment:
Hm, is this a missed find/replace? I see it throughout the PR with seemingly incorrect capitalization.